tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create.shlib

```diff
@@ -119,9 +119,8 @@ function check_feature_set
 	while read line; do
 		typeset flag=1
 
-		if [[ "$line" == "#*" ]]; then
-			continue
-		fi
+		# Skip comments
+		[[ $line = \#* ]] && continue
```

Comment:
This seemed entirely unrelated and belongs in it's own commit.

Reply:
I have created a separate commit for this.

Reply:
That clears things up.  Thanks.